e2e/atlas_alerts_test.go

```diff
@@ -104,8 +104,8 @@ func TestAtlasAlerts(t *testing.T) {
 			t.Fatalf("unexpected error: %v", err)
 		}
 
-		if len(alerts) != 7 {
-			t.Errorf("got=%#v\nwant=%#v\n", len(alerts), 7)
+		if len(alerts) == 0 {
+			t.Errorf("got=%#v\nwant=%#v\n", len(alerts), 14)
```

Comment:
[nit] let's change the message as well
```suggestion
			t.Errorf("got=%#v\nwant>0\n", len(alerts))
```